ui/lib/core/addon/components/form-field.hbs

```diff
@@ -16,22 +16,42 @@
   {{/unless}}
   {{#if @attr.options.possibleValues}}
     {{#if (eq @attr.options.editType "radio")}}
-      <div class="control columns" data-test-radio-input>
+      <div class="control {{unless this.hasRadioSubTextAndLabel 'columns'}}" data-test-radio-input>
```

Comment:
radio buttons are required to have a label, so I think this can be shortened to `hasLabelSubtext` or `hasRadioSubtext` (I think it's already clear that the subtext pertains to the radio label, not the form field label.)

```suggestion
      <div class="control {{unless this.hasLabelSubtext 'columns'}}" data-test-radio-input>
```